src/main/java/org/prebid/server/handler/ExceptionHandler.java

```diff
@@ -38,7 +38,7 @@ private static boolean shouldLogException(Throwable exception) {
 
     private static boolean isConnectionResetException(Throwable exception) {
         return exception instanceof IOException
-                && StringUtils.equals("Connection reset by peer", exception.getMessage());
+                && StringUtils.equals("readAddress(..) failed: Connection reset by peer", exception.getMessage());
```

Comment:
@And1sS Is it possible that `Connection reset by peer` can be thrown other method than `readAddress()`?

Reply:
We cannot trace all the methods that may throw the `Connection reset by peer error`. Therefore, I suggest sticking with Oleksandr's implementation and modifying it to use the `contains` method instead of `equals` if necessary(Or, probably, just add another variants of string for speed).

Reply:
Our only concern right now is the flood of useless "readAddress(..) failed: Connection reset by peer" messages. I'd stick with `equals` here just be a bit lighter on the CPU.